packages/cozy-sharing/locales/fr.json

```diff
@@ -40,7 +40,9 @@
     },
     "twoStepsConfirmation": {
       "confirm": "Confirmer",
-      "reject": "Refuser"
+      "reject": "Refuser",
+      "oneContactToConfirm": "1 contact en attente de confirmation. Confirmer son identité pour lui permettre d’accéder aux éléments que vous avez partagé avec lui.",
+      "multipleContactsToConfirm": "%{numberOfContacts} contacts en attente de confirmation. Confirmer leur identité pour leur permettre d’accéder aux éléments que vous avez partagé avec eux."
```

Comment:
t() is pretty good, it's already deal with single / multiple stuff. 

https://github.com/airbnb/polyglot.js#pluralization 

So you can do: 
```jsx
t(`Share.twoStepsConfirmation.contactsToConfirm`, {
      smart_count: recipientsToBeConfirmed.length
    })
... 
```
```json
contactsToConfirm: "%{smart_count} contact en attente de confirmation. Confirmer son identité pour lui permettre d’accéder aux éléments que vous avez partagé avec lui. |||| %{smart_count} contacts en attente de confirmation. Confirmer leur identité pour leur permettre d’accéder aux éléments que vous avez partagé avec eux."

Reply:
Fixed, thanks for the tip